central/scannerdefinitions/handler/handler.go

```diff
@@ -121,7 +124,10 @@ func (h *httpHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 func (h *httpHandler) get(w http.ResponseWriter, r *http.Request) {
 	if fileType := r.URL.Query().Get("type"); fileType != "" {
 		if v4FileName, exists := v4FileMapping[fileType]; exists {
-			h.getMappingFile(w, r, v4FileName)
+			h.getV4Files(w, r, mappingUpdaterKey, v4FileName)
+			return
+		} else if fileType == "cvss" {
+			h.getV4Files(w, r, cvssUpdaterKey, "")
 			return
 		}
```

Comment:
Good catch, I can add that

Reply:
It does because the logic runs out to the previous Scanner V2 definition, right? I think we should be explicit about that scenario and return 404 if `type` (now called `file`) is invalid. Can you please add that around line #133?  It's a simple line:

```golang
writeErrorNotFound(w)
```